src/plugins/media/render.js

```diff
@@ -16,6 +16,22 @@ export default function renderBlock({ node, attributes, isFocused }, editor, nex
             isFocused={isFocused}
             />
 
+    case 'media_alt':
+        return <div
+            className="media-alt"
+            attributes={attributes}
+            >
+                {children}
+            </div>
```

Comment:
Style: closing tag should be aligned to statement, and content indented once. See for example https://github.com/openstax-poland/cnx-designer/blob/master/src/plugins/section/render.js#L10-L12.